app/services/feature_flag.rb

```diff
@@ -32,7 +32,6 @@ def feature
     [:summer_recruitment_banner, 'Show a banner to indicate a shorter recruitment timeframe during summer', 'Richard Pattinson'],
     [:support_user_revert_withdrawn_offer, 'Allows a support user to revert an application withdrawn by the candidate', 'James Glenn'],
     [:draft_vendor_api_specification, 'The specification for Draft Vendor API v1.1', 'Abeer Salameh'],
-    [:change_course_details_before_offer, 'Allows providers to change course choice details before the point of offer', 'James Glenn'],
```

Comment:
could we add a data migration to remove this feature as well from the db?

Reply:
🆗